rancher-config/docker-compose.yml

```diff
@@ -10,6 +10,7 @@ services:
     es-master:
         labels:
             io.rancher.scheduler.global: true
+            io.rancher.scheduler.affinity:host_label: escluster
```

Comment:
Reading the [docs](http://rancher.com/docs/rancher/latest/en/cattle/scheduling/#values) it looks like the value needs to be in the format `key=value`.

Reply:
I don't think that it is needed. I'll check in the dev environment and see what happens.

Reply:
You're right. the label does need to be `key=value`